website/docs/guides/core-to-cloud-1.md

```diff
@@ -119,42 +131,46 @@ This section highlights the development configurations you’ll need for your db
 
 ### dbt Cloud environments
 
-The concept of an [environment](/docs/environments-in-dbt) in dbt Cloud is the same as a `target` in dbt Core.
-
-The primary difference between a dbt Cloud environment and a `target` in dbt Core is that you can make these configurations through the dbt Cloud UI instead of within the `profiles.yml` file.
+The most common data environments are production, staging, and development. The way dbt Core manages [environments](/docs/environments-in-dbt) is through `target`, which are different sets of connection details. 
 
-This difference streamlines the process of switching between development, staging, and production contexts, removing the need to manually edit the `profiles.yml` file. dbt Cloud environments also integrate with additional features such as job scheduling, version control, and more &mdash; making it easier to manage the full lifecycle of your dbt projects within a single platform. You can [set up](/reference/dbt-jinja-functions/target) or [customize](/docs/build/custom-target-names) target names in dbt Cloud.
+[dbt Cloud environments](/docs/dbt-cloud-environments) go further by:
+- Integrating with features such as job scheduling or version control &mdash; making it easier to manage the full lifecycle of your dbt projects within a single platform.
+- Streamlining the process of switching between development, staging, and production contexts.
+- Making it easy to configure environments through the dbt Cloud UI instead of manually editing `profiles.yml` file. You can also [set up](/reference/dbt-jinja-functions/target) or [customize](/docs/build/custom-target-names) target names in dbt Cloud.
```

Comment:
```suggestion
- Making it easy to configure environments through the dbt Cloud UI instead of manually editing the `profiles.yml` file. You can also [set up](/reference/dbt-jinja-functions/target) or [customize](/docs/build/custom-target-names) target names in dbt Cloud.
```